content/docs/capabilities/directory-sync.mdx

```diff
@@ -0,0 +1,54 @@
+---
+id: directory-sync
+title: Directory Sync
+description: Directory Sync in Pomerium Enterprise allows you to import organizational directory data and external data sources you can use in authorization policies.
+keywords: [directory sync, idp, identity provider, groups, directory]
+---
+
+# Directory Sync
+
+**Directory Sync** is the process of synchronizing external directory data from your identity provider into the Enterprise Console. This document discusses how directory sync works in Pomerium and its use cases.
+
+:::enterprise
+
+**Directory Sync** is a Pomerium Enterprise feature. [Contact us](https://www.pomerium.com/enterprise-sales/) to upgrade today.
+
+:::
+
+## Directory sync in the Enterprise Console
+
+To start a directory sync in the Enterprise Console:
+
+1. Go to the **Identity Providers** tab
+1. Select your **Identity Provider**
+1. Next to [**IDP Options**](#idp-options), fill out the required fields
+1. In the [**Polling Min Delay**](#monitor-directory-sync) and [**Polling Max Delay**](#monitor-directory-sync) fields, enter the appropriate durations
```

Comment:
How would I choose appropriate durations? If there are reasonable defaults for these settings, maybe we could leave this step out of the main instructions?

Reply:
You make a good point, but we did have a ticket awhile back where users were unaware their sync wasn't completing. It depends on the size of your directory, so the defaults don't always work. You have to monitor the sync to ensure it completes, which may require adjusting min/max polling settings to get the sync right.